setup.py

```diff
@@ -0,0 +1,34 @@
+import distutils.core
+from distutils.command.install import install
+import subprocess
+
+
+class PyenvInstall(install):
+    def run(self):
+        print(subprocess.check_output(['bash', 'bin/pyenv-installer']))
+
+with open('README.rst') as file:
+    long_description = file.read()
+
+distutils.core.setup(
+    version='20150113',
```

Comment:
Yes ... I started thinking about this already. My first aim was to get something out on PyPi to get rid of the wrong picture that pyenv is an abandoned preject as it is clearly not :) Next steps for me would be to clarify the pyenv, pyenv-installer diecrepancy in the documentation and find some workable way to straighten out the versioning.

The pyenv-installer needs it's own version, but if you install pyenv via pip then the actually installed version of pyenv should show up in pip freeze IMO ... so this is not trivial, but it's on my radar already.